health_gis/templates/health_dash/health.html

```diff
@@ -697,6 +719,32 @@ <h6>COVID-19 Lockdown Zones</h6>
         mymap.removeLayer(activatedHealthFacility[facility]);
 };
 
+function removewards(){

```

Comment:
remove useless comments
```suggestion
function removewards(){
//  var dist = cql;
        //  console.log(wardLayerList);
        //  var wms_layer = L.tileLayer.wms('https://makerghat.urbansciences.in/geoserver/wms',{
        //     layers: 'geonode:corporatorwardboundary_mumbai_13may',
        //     format: 'image/png',
        //     transparent:'true',
        //     // version: '1.3.0',
        //     cql_filter:"district='"+dist+"'",
        //     style:""});
        //   wardLayerList.forEach((l,index) => {
        //   fil = (l.options.cql_filter);
        //   if(dist === fil){
        //         mymap.removeLayer(l);
        //         wardLayerList.splice(index, 1);
        //   }

        // });
```

Reply:
done